app/build.gradle

```diff
@@ -102,7 +102,14 @@ dependencies {
 
     //swipe_layout
     implementation 'com.daimajia.swipelayout:library:1.2.0@aar'
+
+    //Room
+    def room_version= '2.2.3'
```

Comment:
versions should be specified in `gradle.properteries` and be in CONSTANT_CASE

Reply:
I could not find any such thing in the developer docs, based on the project scope it could be in the project's build.gradle or app's build.gradle, can you share relevant doc links for the same

Reply:
I am following the example of the project. I don't see any other dependency declared like this but I do see leakcanary, dagger, kotlin & butterknife declared in gradle.properties.

It also just so happens that gradle.properties is the best place to put them, particularly in a multi module project.

Reply:
@seannemann21 AFAIK, for multi-module projects, project-level build.gradle should be the place for such things

Reply:
No, best practise advice is to configure your projects using a precompiled script or a plugin in your buildsrc folder. The best place to keep your version numbers is in gradle.properties, buildsrc is a bit annoying becuse any change in there cause a recompilation of the entire project.
[Use buildsrc](https://medium.com/google-developer-experts/19-tips-for-gradle-in-android-projects-2019-edition-11af704eb06e)
[Proper versioning and great library](https://github.com/jmfayard/refreshVersions)

Reply:
Okay, well that would need pulling out other dependencies from project build.gradle and app build.gradle as well, right?, Should we not create a separate issue for the same and take it from there?

Reply:
Perhaps, but I am reviewing this addition. If you define a version put it in gradle.properties with the rest of them, as is evidently project style.